### PR TITLE
radare2: 0.9.7 -> 0.10.2

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Still WIP. Might solve #15269

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
I added a new package: `libuna` which is (or seems to be) required by `radare2`. I don't know why it does not build yet, help is appreciated!

File: pkgs/development/libraries/libuna/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+stdenv.mkDerivation rec {
+  name = "libuna";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libuna/archive/20150927.tar.gz";
+    sha256 = "0dyvsrbwy6hd6yli57rqpl4q1kfvvs40fvrm4ngna8fnhgvykvlb";
+  };
+
+  buildInputs = [ autoconf automake libtool dos2unix ];
+  nativeBuildInputs = [ pkgconfig ];
```
Comment (joachifm) [inline pkgs/development/libraries/libuna/default.nix]: Not that it matters, but autotools are typically included in the native build tools as well.

Comment (matthiasbeyer) [inline pkgs/development/libraries/libuna/default.nix]: It failed calling them without them included... I'm not sure why.

Comment (joachifm) [inline pkgs/development/libraries/libuna/default.nix]: It failed with them in `nativeBuildInputs`? That is odd, in the case where the target system is the same as the build system, all `buildInputs` end up in `nativeBuildInputs`, afaik.

Comment (joachifm) [inline pkgs/development/libraries/libuna/default.nix]: Anyway, doesn't matter ...

**Comment by joachifm** [comment]
Looks to me like nox is having some trouble creating a checkout, it can be a lil' particular, never looked into why. Has this been rebased on current HEAD?

**Comment by matthiasbeyer** [comment]
Rebased.

**Comment by Mic92** [comment]
merged in 081c5494acd16369749604d4328595c81eb8e85b
libuna was not required to build radare2

**Pull request closed without merging**